service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwinDevice.java

```diff
@@ -644,10 +644,21 @@ private TwinCollection setToMap(Set<Pair> set)
         return map;
     }
 
+    /**
+     * Getter to get model Id
```

Comment:
Is this kind of language typical? "Getter to get" and "Setter to set" seems redundant.

Reply:
Yes that's the language I have seen for other get and set method.

Reply:
addressed this.

Reply:
Then the other places should be updated. This isn't a good description. "Gets the model Id" is much better